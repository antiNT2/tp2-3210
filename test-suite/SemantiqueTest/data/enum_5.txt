num A;
enum A {
    B, C, D, d
}
A d;