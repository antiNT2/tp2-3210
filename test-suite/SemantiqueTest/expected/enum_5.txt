Identifier A has multiple declarations